tests/test_clients/test_methods/test_base/test_empty_error_handlers.py

```diff
@@ -0,0 +1,13 @@
+from botx.clients.methods.base import BotXMethod
+
+
+class TestMethod(BotXMethod):
+    __url__ = "/path/to/example"
+    __method__ = "GET"
+    __returning__ = str
+
+
+def test_method_empty_error_handlers():
```

Comment:
Nit: Move the class inside the function.

Reply:
mypy raised internal error when I moved

Reply:
Hmm, ok.